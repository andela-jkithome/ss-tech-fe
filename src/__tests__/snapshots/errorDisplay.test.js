import React from 'react';
import renderer from 'react-test-renderer';
import ErrorDisplay from '../../components/errorDisplay';

it('Renders correctly', () => {
  const component = renderer.create(
    <ErrorDisplay
      header="Error Fetching Fact!"
      text="There was an error fetching the fact. Please try again"
    />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
