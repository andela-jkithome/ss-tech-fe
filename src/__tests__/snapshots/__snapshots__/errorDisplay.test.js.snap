// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="column"
  >
    <div
      className="ui negative message jokemessage"
    >
      <div
        className="header"
      >
        Error Fetching Fact!
      </div>
      <p>
        There was an error fetching the fact. Please try again
      </p>
    </div>
  </div>
</div>
`;
